public/app/features/alerting/unified/components/rules/RulesFilter.tsx

```diff
@@ -13,10 +13,15 @@ import { Stack } from '@grafana/experimental';
 
 const ViewOptions: SelectableValue[] = [
   {
-    icon: 'folder',
-    label: 'Groups',
+    icon: 'list-ul',
+    label: 'List',
     value: 'group',
```

Comment:
Can we change the `group` to `list`? Or we should keep it due to compatibility reasons?